Description: update readme
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Last-Update: 2024-03-14
Applied-Upstream: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
 unlzx
 =====
 
-unlzx is a program to extract LZX archives on Haiku
+unlzx is a program to extract LZX archives created on Amiga OS and on Haiku
